com.unity.render-pipelines.universal/Documentation~/Building-For-Consoles.md

```diff
@@ -1,16 +1,10 @@
-# Building your Project for consoles
+# Building your Project for Closed platforms
 
-To build a Project for the **PlayStation 4**, **PlayStation 5**, **Xbox Series**, **Game Core Xbox One** or **Xbox One**, you need to install an additional package for each platform you want to support. The packages for each platform are:
-
-- **PlayStation 4**: com.unity.render-pipelines.ps4
-- **PlayStation 5**: com.unity.render-pipelines.ps5
-- **Xbox One**: com.unity.render-pipelines.xboxone
-- **Game Core Xbox Series**: com.unity.render-pipelines.gamecore
-- **Game Core Xbox One**: com.unity.render-pipelines.gamecore
+If you have a license to develop games for Closed platforms that require you to meet the confidentiality and legal agreements of the platform provider, then see the relevant developer forums for a link to the console specific render pipeline package.
 
 ## Platform package installation
 
-To install a platform package, download it from the relevant platform developer forum (you cannot get these packages from the package registry or Package Manager). To do this:
+To install a platform package, download it from the relevant platform developer forum. Note that you cannot get these packages from the package registry or Package Manager. To do this:
```

Comment:
My suggestion:

Closed platform packages are not available in the package registry and the Package Manager.

To install a closed platform package:

1. Download the package from the relevant platform developer forum.

2. Use the Package Manager to install the package locally. For information on how to install packages locally, see [Installing a local package](https://docs.unity3d.com/Manual/upm-ui-local.html).

Reply:
Good point -- I'll implement your suggestions.